src/pages/Voting/components/AddressesList.tsx

```diff
@@ -60,16 +65,119 @@ function AddressVotingState({
   );
 }
 
+const STAKE_POOL_ADDRESS_BY_VOTER_QUERY = gql`
+  query current_staking_pool_voter($voter_address: String) {
+    current_staking_pool_voter(where: {voter_address: {_eq: $voter_address}}) {
+      staking_pool_address
+    }
+  }
+`;
+
+type CurrentStakingPoolVoter = {
+  current_staking_pool_voter: [
+    {
+      __typename: string;
+      staking_pool_address: string;
+    },
+  ];
+};
+
 export default function AddressesList({
-  addressVoteMap,
   proposal,
+  accountAddress,
 }: AddressesListProps) {
+  const [state, _] = useGlobalState();
+  const [mapLoading, setMapLoading] = useState<boolean>(false);
+  const [addressVoteMap, setAddressVoteMap] = useState<AddressToVoteMap[]>();
+
+  const {loading, error, data} = useGraphqlQuery(
+    STAKE_POOL_ADDRESS_BY_VOTER_QUERY,
+    {
+      variables: {
+        voter_address:
+          "0x8ac6fe79b3656feda6485fcf872e00033dd35cd9d8e10b485e3dee34949b2f47",
```

Comment:
What is this address? Let's extract it to an environment variable with a descriptive name. Does it also need to change based on the selected network?

Reply:
good catch! it should have been a variable, updated!